### PR TITLE
Lrn

### PR DESCRIPTION
**Pull request by hdmetor**
First draft for https://github.com/pytorch/pytorch/issues/653

### CODE REVIEW

File: torch/nn/functional.py [hunk 1]
```diff
@@ -607,6 +607,10 @@ def sigmoid(input):
     return _autograd_functions.Sigmoid.apply(input)
 
 
+def spatial_cross_map_lrn(input, size, alpha, beta, k):
+  return _functions.SpaTialCrossMapLRN.apply(input, size, alpha, beta, k)
```
Comment (Kaixhin) [inline torch/nn/functional.py]: You've got a typo here, so the import should be failing.

**Comment by TaihuLight** [comment]
@Kaixhin @hdmetor (1) How to solve the failed check? (2) How to add the LRN function in code writed with Pytorch 0.3?

**Comment by hdmetor** [comment]
Thanks @Kaixhin 
corrected

**Comment by soumith** [comment]
closed in favor of https://github.com/pytorch/pytorch/pull/4237

**Comment by hdmetor** [comment]
@soumith you closed this PR in favor of itself? 🤔

**Comment by soumith** [comment]
my bad, i meant #4667 , sorry :)

**Pull request closed without merging**